### DISCUSSION TITLE
Upcoming update of page header visual treatment

### DISCUSSION THREAD

**Discussion by fralongo** [cloudscape-design/components#2235] [Announcements]
Based on customer feedback, Cloudscape has decided to update the page header default color treatment from dark to light, and add styling capabilities to customize page headers in specific use cases. We will release these changes at end of May 2024.

Your application will automatically benefit from the update. [Contact us](https://github.com/cloudscape-design/components/discussions) if you have any question on the update.

**Comment by brockuniera** [comment]
Did this change release yet?

**Reply by fralongo** [reply]
@brockuniera yes, we released the change in https://github.com/cloudscape-design/components/releases/tag/3.0.639. With this changes, we also released new capabilities in the [content layout component](https://cloudscape.design/components/content-layout/), updates to our [layout article](https://cloudscape.design/foundation/visual-foundation/layout/) and (hero header pattern)[https://cloudscape.design/patterns/general/hero-header/].
